new_project/examples/separate_apps/src/maker.ts

```diff
@@ -0,0 +1,84 @@
+import { formatEther } from "ethers/utils";
+import readLineSync from "readline-sync";
+import { toBitcoin } from "satoshi-bitcoin-ts";
+import { checkEnvFile, startClient } from "./lib";
+
+(async function main() {
+    checkEnvFile(process.env.DOTENV_CONFIG_PATH!);
+
+    const maker = await startClient(0);
+
+    console.log(
+        "[Maker] Bitcoin balance: %f. Ether balance: %f",
+        parseFloat(await maker.bitcoinWallet.getBalance()).toFixed(2),
+        parseFloat(
+            formatEther(await maker.ethereumWallet.getBalance())
+        ).toFixed(2)
+    );
+
+    const peerId = await maker.comitClient.getPeerId();
+    const addressHint = await maker.comitClient
+        .getPeerListenAddresses()
+        .then(addresses => addresses[0]);
+
+    console.log("[Maker] peerId:", peerId);
```

Comment:
:wrench: For consistency with respect to the other file.
```suggestion
    console.log("[Maker] Peer Id:", peerId);
```